scarlet/source.py

```diff
@@ -1,16 +1,17 @@
 from functools import partial
```

Comment:
`partial` is no longer used in this module, please remove.

Reply:
lines 45 and 282

Reply:
Ah, I see what's happening. PyCharm was telling me that these (and other) imports were unused because they used the wildcard. So if any of the modules that you `import *` from already have `partial` imported, then python just uses those and overwrites the one already defined in the model. Another reason to not use wildcard imports... or at least define `__all__` in all of our moduals.